contracts/src/v0.8/vendor/BufferChainlink.sol

```diff
@@ -173,11 +173,13 @@ library BufferChainlink {
     }
 
     // Copy remaining bytes
-    uint mask = 256 ** (32 - len) - 1;
-    assembly {
-      let srcpart := and(mload(src), not(mask))
-      let destpart := and(mload(dest), mask)
-      mstore(dest, or(destpart, srcpart))
+    unchecked {
```

Comment:
Updated from the latest [Buffer.sol](https://github.com/ensdomains/buffer/blob/master/contracts/Buffer.sol), as it relies on over/underflow.

Reply:
We can... but that would mean defining an empty `uint max` outside the `unchecked` block so that the statements that use it in the `assembly` block pass.

Also, since we're [importing this from ENS](https://github.com/ensdomains/buffer/blob/master/contracts/Buffer.sol) into `vendor/`, I'd rather keep it exactly the same as their implementation.